core/src/main/resources/lib/hudson/help.jelly

```diff
@@ -24,19 +24,17 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
   <st:documentation>
-    Icon linked to the specified @href
+    Displays a help button
 
     <st:attribute name="href" use="required">
       Destination of the help icon's link
     </st:attribute>
     <st:attribute name="tooltip">
       The tooltip to display
     </st:attribute>
-    <st:attribute name="iconSize">
-      Icon size, available are: small, medium, large, xlarge
-    </st:attribute>
   </st:documentation>
-  <a href="${attrs.href}" class="jenkins-help-button ${attrs.iconSize}" tooltip="${attrs.tooltip}">
-    ?
+
+  <a href="${attrs.href}" class="jenkins-button" tooltip="${attrs.tooltip ?: '%Help'}">
```

Comment:
I know it's not directly related, but not opening external links in a new tab seems user hostile (and if they're not external, why is it not a normal inline help link? -- projectRelationship-help probably should be updated to not be weird)

```suggestion
  <a href="${attrs.href}" class="jenkins-button" tooltip="${attrs.tooltip ?: '%Help'}" target="_blank" rel="noopener noreferrer">
```

Happy to file an independent PR though if this goes beyond scope.

Reply:
That's a great suggestion - cheers!